public/css/reset.css

```diff
@@ -0,0 +1,9 @@
+
+  *{
+	margin: 0;
+	padding: 0;
+	list-style: none;
+	text-decoration: none;
+	border: none;
+	outline: none;
```

Comment:
cool. this is a good practice to reset the CSS. also, there is a pre-configured CSS for resetting all default styles: https://meyerweb.com/eric/tools/css/reset/

Reply:
Implemented 🚀